src/vision/src/ipm_filter.cpp

```diff
@@ -0,0 +1,20 @@
+/*
+ * Created By: Robyn Castro
+ * Created On: June 15, 2017
+ * Description: Filters out all non-green from the image
+ *              then publishes the new filtered image.
+ */
+
+#include "../include/IPMFilterNode.h"
+#include <ros/ros.h>
+
+int main(int argc, char** argv) {
+    // Setup your ROS node
+    std::string node_name = "IPMFilter";
```

Comment:
Just realized hsv_filter node_name is in snake case. For consistency sake, should IPMFilter be changed or hsv_filter?

Reply:
Good point. I'm not sure if we've every explicitly looked at this and decided before, but most nodes, including the sample node, seem to be in snake case. I've changed this here, as well as in `vision/CMakeLists.txt`.